src/modules/EventModule.cpp

```diff
@@ -0,0 +1,792 @@
+									if (luckyTicketId == userTicket)
+									{
+										config.luckyUser = user.m_UserId;
+									}
+								}
+							}
+
+							//Clear all the tickets
+							for (auto& user : raffleDrawGuildList[channel.guildId()])
+								user.m_TicketIds.clear();
+
+							//Set the reward claiming time (24 hours)
+							config.raffleDrawRewardClaimTimer = new QTimer;
+							config.raffleDrawRewardClaimTimer->setInterval(24 * 3600000);
+							config.raffleDrawRewardClaimTimer->setSingleShot(true);
+
+							snowflake_t guildID = channel.guildId();
+							snowflake_t chan = channel.id();
+							snowflake_t authorId = message.author().id();
+							QObject::connect(config.raffleDrawRewardClaimTimer, &QTimer::timeout, [this, guildID, chan, authorId]()
+							{
+								auto& config = getServerEventData(guildID);
+								config.claimedReward = true;
+								UmikoBot::Instance().createMessage(chan, "The owner of the price in the RaffleDraw didn't get their rewards!\n"
+																		"The role(s) who have permission can start this event again!");
+							});
+							config.raffleDrawRewardClaimTimer->start();
+							config.isEventRunning = false;
+							config.eventRaffleDrawRunning = false;
+							config.eventTimer->stop(); // Forcefully stop the timer.
+							UmikoBot::Instance().createMessage(message.channelId(), embed);
+						}
+					}
+
+				});
+	});
+	RegisterCommand(Commands::EVENT_GET_REWARD, "getreward", [this](Client& client, const Message& message, const Channel& channel)
+	{
+		QStringList args = message.content().split(' ');
+		if (args.size() != 1)
+		{
+			client.createMessage(message.channelId(), "**Wrong Usage of Command!** ");
+			return;
+		}
+		auto& config = getServerEventData(channel.guildId());
+		if (config.isEventRunning)
+		{
+			client.createMessage(message.channelId(), "**You can't get free rewards anytime!**");
+			return;
+		}
+		auto& configRD = raffleDrawGuildList[channel.guildId()][raffleDrawGetUserIndex(channel.guildId(), message.author().id())];
+		if (!config.claimedReward)
+		{
+			if(message.author().id() == config.luckyUser)
+			{
+				if (config.raffleDrawRewardClaimTimer->isActive())
+				{
+					CurrencyModule* currencyModule = static_cast<CurrencyModule*>(UmikoBot::Instance().GetModuleByName("currency"));
+					auto& currencyConfig = currencyModule->getServerData(channel.guildId());
+					auto& authorCurrency = currencyModule->guildList[channel.guildId()][currencyModule->getUserIndex(channel.guildId(), message.author().id())];
+					QString authorName = UmikoBot::Instance().GetName(channel.guildId(), message.author().id());
+					config.claimedReward = true;
+					Embed embed;
+					embed.setColor(15844367);
+					authorCurrency.setCurrency(authorCurrency.currency() + 500);
+					embed.setTitle("**Raffle Draw Reward goes to " + authorName + "!**");
+					embed.setDescription("Congratulations **" + authorName + "**!\n"
+										"You just got `500` " + currencyConfig.currencySymbol);
```

Comment:
In almost all the other places the symbol is included inside the code block, so would be better if we did that here as well :)